velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -141,6 +142,22 @@ Writer::Writer(
   } else {
     flushPolicy_ = std::make_unique<DefaultFlushPolicy>();
   }
+
+  if (options.schema) {
+    ArrowSchema arrowSchema;
+    auto leafPool = pool_->addLeafChild("parquet-write-schema-convert");
+    exportToArrow(
```

Comment:
Why do we need to create a vector to convert TypePtr to Arrow schema? There should be a way to convert directly.

Reply:
There is no API currently in the Arrow Bridge. I think we can extract this logic though.
@JkSelf can we extract the schema logic?